script.py

```diff
@@ -1,13 +1,38 @@
 import os
-from datetime import datetime
+import requests
 
 # Get input from environment variables
+website = os.getenv("INPUT_WEBSITE", "")
```

Comment:
so these ENV vars will be set by github based on the action.yml definition above?
Can you add a comment?

Reply:
Added